tsdef/RemoteTrackPublication.d.ts

```diff
@@ -0,0 +1,19 @@
+import { Track } from './Track';
+import { TrackPublication } from './TrackPublication';
+
+export class RemoteTrackPublication extends TrackPublication {
+  isSubscribed: boolean;
+  isTrackEnabled: boolean;
+  kind: Track.Kind;
```

Comment:
Is `Track.Kind` defined  in `Track.d.ts`?

Reply:
Yes. There is a `namespace Track` that defines `Kind`. The actual class `Track` is `Track.Kind` from the namespace.